src/NumFieldOrd/NfOrd/Elem.jl

```diff
@@ -256,21 +255,21 @@ end
 ################################################################################
 
 @doc Markdown.doc"""
-    discriminant(B::Array{NfAbsOrdElem, 1}) -> fmpz
+    discriminant(B::Array{NumFieldOrdElem, 1}) -> fmpz
 
 Returns the discriminant of the family $B$.
 """
-function discriminant(B::Array{NfAbsOrdElem{S, T}, 1}) where {S, T}
-  length(B) == 0 && error("Number of elements must be non-zero")
-  length(B) != degree(parent(B[1])) &&
-        error("Number of elements must be $(degree(parent(B[1])))")
+function discriminant(B::Vector{T}) where T <: NumFieldOrdElem
```

Comment:
Isn't this `absolute_discriminant`?

Reply:
Well, textbooks define the discriminant of n algebraic numbers this way, where n is the degree over Q. I can change it if you want...